src/abft-messages-seed.md

```diff
@@ -1,68 +1,85 @@
+\newcommand \DigestLookup {\mathrm{DigestLookup}}
+$$
+
 # Seed
 
-Informally, the protocol interleaves \\(\delta_s\\) seeds in an alternating
-sequence. Each seed is derived from a seed \\(\delta_s\\) rounds in the past through
-either a hash function or through a VRF, keyed on the entry
-proposer. Additionally, every \\(\delta_s\delta_r\\) rounds, the digest of a previous entry
-(specifically, from round \\(r-\delta_s\delta_r\\)) is hashed into the result. The seed
-proof is the corresponding VRF proof, or 0 if the VRF was not used.
+Informally, the protocol interleaves \\( \delta_s \\) seeds in an alternating
+sequence. Each seed is derived from a seed \\( \delta_s \\) rounds in the past through
+either a hash function or through a \\( \VRF \\), keyed on the entry
+proposer. Additionally, every \\( \delta_s\delta_r \\) rounds, the digest of a previous
+entry (specifically, from round \\( r-\delta_s\delta_r \\)) is hashed into the result.
+The seed proof is the corresponding VRF proof, or 0 if the \\( \VRF \\) was not used.
 
-> This section makes use of the \\(VRF.Prove(.)\\), \\(VRF.ProofToHash(.)\\) and
-> \\(VRF.Verify(.)\\) cryptographic low-leve functions defined in the reference
-> implementation's Algorand Sodium Fork. They respectively output a \\(VRF\\) proof
-> and a \\(VRF\\) hash for a given input, and perform verification of a \\(VRF\\)
-> run. For details on the input structure and inner workings, please refer to the
-> Appendix A in the [Algorand Crypto Specification](./crypto.md).
+> This section makes use of the \\( \VRF.\Prove(.) \\), \\( \VRF.\ProofToHash(.) \\)
+> and \\( \VRF.\Verify(.) \\) cryptographic low-leve functions defined in the reference
+> implementation's Algorand Sodium Fork. They respectively output a \\( \VRF \\)
+> proof and a \\( \VRF \\) hash for a given input, and perform verification of a
+> \\( \VRF \\) run. For details on the input structure and inner workings, please
+> refer to the Appendix A in the [Algorand Crypto Specification](./crypto.md).
 
-More formally, suppose \\(I\\) is a correct proposer in round \\(r\\) and period
-\\(p\\).
+More formally, suppose \\( I \\) is a correct proposer in round \\( r \\) and period
+\\( p \\).
 
 Let
 
-- \\((pk, B, r_{fv}, r_{lv}) = Record(L, r - \delta_b, I)\\),
+- \\( (\pk, B, r_\fv, r_\lv) = \Record(L, r - \delta_b, I) \\),
 
-- \\(sk\\) be the secret key corresponding to \\(pk\\),
+- \\( \sk \\) be the secret key corresponding to \\( \pk \\),
 
-- \\(\alpha\\) be a 256-bit integer.
+- \\( \alpha \\) be a 256-bit integer.
 
-Then \\(I\\) computes the seed proof \\(y\\) for a new entry as follows:
+Then \\( I \\) computes the seed proof \\( y \\) for a new entry as follows:
 
-- If \\(p = 0\\):
-  - \\(y = VRF.Prove(Seed(L, r-\delta_s), sk)\\),
-  - \\(\alpha = Hash(I || VRF.ProofToHash(y))\\).
+- If \\( p = 0 \\):
+  - \\( y = \VRF.\Prove(\Seed(L, r-\delta_s), \sk) \\),
+  - \\( \alpha = \Hash(I || \VRF.\ProofToHash(y)) \\).
```

Comment:
@cce @jannotti the order of the hash arguments was adapted to https://github.com/algorand/go-algorand/blob/b6e5bcadf0ad3861d4805c51cbf3f695c38a93b7/agreement/proposal.go#L130 and https://github.com/algorand/go-algorand/blob/b6e5bcadf0ad3861d4805c51cbf3f695c38a93b7/agreement/proposal.go#L178